examples/reference/panes/HoloViews.ipynb

```diff
@@ -6,41 +6,70 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import holoviews as hv\n",
     "import panel as pn\n",
-    "pn.extension('plotly')"
+    "\n",
+    "hv.extension(\"bokeh\")\n",
```

Comment:
When I changed the notebook examples suddenly they started complaining that I needed to run `hv.extension(...)`. So I did. But I don't know why its needed now and not before??

Reply:
Changed it to load both extensions with `hv.extension()`